go/vt/vtctl/grpcvtctldserver/query.go

```diff
@@ -150,6 +150,7 @@ func rowToSchemaMigration(row sqltypes.RowNamedValues) (sm *vtctldatapb.SchemaMi
 	sm.VitessLivenessIndicator = row.AsInt64("vitess_liveness_indicator", 0)
 	sm.UserThrottleRatio = float32(row.AsFloat64("user_throttle_ratio", 0))
 	sm.SpecialPlan = row.AsString("special_plan", "")
+	sm.DependentMigrations = row.AsString("dependent_migrations", "")
```

Comment:
I wanted to make this a `[]string` but our marshalling libraries don't like it. See later comment for more context

Ideas appreciated